src/cli.js

```diff
@@ -129,66 +129,71 @@ function help(options = CLIOPTIONS) {
 /**
  * Handle exiting of program
  *
- * @param {null}    exiting - null for bind
- * @param {object}  error   - Object to distinguish between closing events
+ * @param {null}    _       - null for bind
+ * @param {object}  error   - The return code
```

Comment:
Is this an accurate comment given it's not an object? (from what I can see, it used to be).

Reply:
very true! fixing now, nice catch!